app/src/main/java/ch/epfl/skysync/screens/crewpilot/Flight.kt

```diff
@@ -57,6 +57,12 @@ import com.google.maps.android.compose.MarkerState
 import com.google.maps.android.compose.Polyline
 import com.google.maps.android.compose.rememberCameraPositionState
 
+/**
+ * This composable function renders a marker on the map at the given location.
+ *
+ * @param location The location to render the marker at.
+ * @param user The user associated with the location.
+ */
 @Composable
 fun UserLocationMarker(location: Location, user: User) {

```

Comment:
This function is a duplication of the function in `app/src/main/java/ch/epfl/skysync/screens/admin/AdminFlight.kt`

Reply:
Yes, I'll let that to Rafael who's task is to clean the composables code.